src/main/java/spring/service/mapper/UserMapper.java

```diff
@@ -0,0 +1,24 @@
+package spring.service.mapper;
+
+import org.springframework.stereotype.Component;
+import spring.dto.UserResponseDto;
+import spring.model.User;
+
+@Component
+public class UserMapper {
+    public UserResponseDto convertUserToDto(User user) {
+        UserResponseDto userResponseDto = new UserResponseDto();
+        userResponseDto.setName(user.getName());
+        userResponseDto.setSurname(user.getSurname());
+        userResponseDto.setEmail(user.getEmail());
+        return userResponseDto;
+    }
+
+    public User convertToUser(UserResponseDto userResponseDto) {
```

Comment:
where do we use this method? Let's remove it)

Reply:
removed